db/migrate/20141229085904_add_attachment_profile_picture_to_users.rb

```diff
@@ -1,11 +1,17 @@
 class AddAttachmentProfilePictureToUsers < ActiveRecord::Migration[4.2]
   def self.up
     change_table :users do |t|
-      t.attachment :profile_picture
+      t.string :profile_picture_file_name
+      t.string :profile_picture_content_type
+      t.bigint :profile_picture_file_size
```

Comment:
This was a `t.integer` before -- please change it (and create a migration for that if it really should become a `bigint`)